### ISSUE TITLE
discussion: Should we have this as an Organization Level README.md?

### ISSUE THREAD

**Issue by sandipndev** [GaloyMoney/awesome-galoy#1]
I have seen many organisations use a `.github` repository inside of which they store a README.md file which explains the outlook and repositories of the organisation that would be helpful to anyone who is visiting the GitHub Organization page for the first time. This gives a sense of direction as to what is where in brief and I think this repository's sole purpose is the same. Any thoughts?

Example: http://github.com/lightningdevkit

**Comment by vindard** [comment]
That's a great idea for visibility I think

**Comment by moksha256** [comment]
This is cool. Seems like a great way to summarize Galoy's purpose while pointing out key repositories and what they do in order to help orient new people exploring GitHub. 

This repository already does that, but people have to _find_ this repository first, so this org-wide README seems better.

Having said that, I'm not a fan of the way lightningdevkit does it. The "Hey there" title doesn't achieve anything and that picture is too big and bold. GitHub isn't a place I want to see that kind of marketing flash...just my opinion.

**Comment by sandipndev** [comment]
> The "Hey there" title doesn't achieve anything and that picture is too big and bold.

Agreed. In my opinion, just a banner on top followed by necessary details like pointing to the most important repositories, how can someone get running their own Galoy stack running and Community Slack links would be the best to keep. Brief, but information rich!

That said, in my opinion, the README.md on this repository is a little longer than what should probably be on the top of organization's GitHub page.

**Comment by moksha256** [comment]
Agreed on all points. I'll come up with a draft README we can show others for feedback.
